src/styles/NavBar.css

```diff
@@ -61,8 +67,41 @@ span {
   color: var(--pp-selected-gray);
 }
 
+.nav-right-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.nav-button {
+  background-color: #ffffff;
+  color: var(--ks-font-medium);
```

Comment:
I would ping @spring on this because color styles arent exactly clear for type; I'm likely sure --ks-font-medium needs to be updated.